datumaro/components/operations.py

```diff
@@ -145,14 +149,44 @@ def _merge_attrs(a: Dict[str, Any], b: Dict[str, Any],
 
         return merged
 
-    @staticmethod
-    def _merge_images(item_a: DatasetItem, item_b: DatasetItem) -> Image:
-        image = None
+    @classmethod
+    def _merge_media(cls, item_a: DatasetItem, item_b: DatasetItem) \
+            -> Union[ByteImage, Image, PointCloud, Video, VideoFrame]:
+        if isinstance(item_a.media, (ByteImage, Image, VideoFrame)) or \
+                isinstance(item_b.media, (ByteImage, Image, VideoFrame)):
```

Comment:
```suggestion
        if isinstance(item_a.media, Image) or \
                isinstance(item_b.media, Image):
```
They all are Image subclasses

Reply:
Fixed